tendrl/commons/message.py

```diff
@@ -153,7 +153,7 @@ def serialize_message(obj):
         serial = obj.isoformat()
         return serial
     elif isinstance(obj, Exception):
-        return yaml.safe_dump(obj)
+        return yaml.safe_dump(obj.message)
```

Comment:
may i know how you passed exception in payload, can you please share that line
you have to give like this for example:
try:
    raise ValueError("Some error")  # i am creating value error manually
except ValueError as ex:  # here ex is important (here ex is exception object not string)
    Event(
        ExceptionMessage(
            priority="info" (any priority)
            publisher="" (some publisher name)
            payload={"message": "some exception related message", exception: ex}
        )
    )